pkg/volume/empty_dir/empty_dir.go

```diff
@@ -103,6 +105,7 @@ func (plugin *emptyDirPlugin) newCleanerInternal(volName string, podUID types.UI
 		mountDetector: mountDetector,
 		plugin:        plugin,
 	}
+	ed.AccountingDu.Init(ed.GetPath())
```

Comment:
embedding when you have a really generic name like Init() exposed is bound to cause problems.  Maybe InitAccounting()?